express-server/src/repositories/DocumentsRepository.ts

```diff
@@ -81,64 +81,6 @@ async function getMachineryDocuments(machineryUID: string) {
 
 }
 
-// async function getFilesFromDirectoryRecursive(directoryPath: string, filesDB: Document[]): Promise<FileMapEntry[]> {
```

Comment:
You just removed commented code. LGTM.

My apologies for not having noticed such large chunk of unused code in a previous PR when it was first introduced.